lib/routes/seagullApi.js

```diff
@@ -331,7 +331,11 @@ module.exports = function (crudHandler, userApiClient, gatekeeperClient, metrics
               user.profile = _.result(document, 'detail.profile');
 
               if (_.isEmpty(user.trustorPermissions)) {
-                delete user.profile.patient;
+                if (user.profile) {
+                  delete user.profile.patient;
+                } else {
+                  log.error(`User ${user.userid} does not have a valid profile. Consider investigaing the account.`);
```

Comment:
nit: typo 'investigaing'

Reply:
Gah! Why isn't there a normal spell check in VSCode :p

Reply:
Added one :)